compiler/rustc_interface/src/passes.rs

```diff
@@ -672,6 +697,9 @@ pub fn write_dep_info(tcx: TyCtxt<'_>) {
     // the side-effect of providing a complete set of all
     // accessed files and env vars.
     let _ = tcx.resolver_for_lowering();
+    // Similarly, analysis, codegen and linking should state which environment
+    // variables they depend on here, as those passes are run after this pass,
+    // but we'll need the information when emitting dependency info to Cargo.
```

Comment:
Could you remove this comment until it's actually done?
I'm not sure it will be done in the way that is described here.